hikey.mk

```diff
@@ -420,9 +420,33 @@ nvme-cleaner:
 ################################################################################
 # Flash
 ################################################################################
+define flash_help
+	@read -r -p "1. Connect USB OTG cable, the micro USB cable (press any key)" dummy
+	@read -r -p "2. Connect HiKey to power up (press any key)" dummy
+endef
+
+.PHONY: recovery
+recovery:
+	@echo "Enter recovery mode to flash a new bootloader"
+	@echo "Jumper 1-2: Closed (Auto power up)"
+	@echo "       3-4: Closed (Boot Select)"
+	@echo "       5-6: Open   (GPIO3-1)"
```

Comment:
I know the getting started and user guide both says GPIO3-1, but for our purposes here, isn't it more descriptive to say or at least add the wording 'Fastboot Mode'? Also for Boot Select, maybe say or add 'Recovery Mode'?